core/src/main/java/org/apache/accumulo/core/clientImpl/InstanceOperationsImpl.java

```diff
@@ -344,9 +433,19 @@ public List<ActiveCompaction> getActiveCompactions()
   }
 
   @Override
+  @Deprecated
   public void ping(String tserver) throws AccumuloException {
-    try (
-        TTransport transport = createTransport(AddressUtil.parseAddress(tserver, false), context)) {
+    HostAndPort hp = HostAndPort.fromString(tserver);
+    ping(new TabletServer(hp.getHost(), hp.getPort()));
+  }
+
+  @Override
+  public void ping(Server server) throws AccumuloException {
+    @SuppressWarnings("unused")
+    var unused = Objects.nonNull(server);
+    Preconditions.checkArgument(server.getType() == ServerType.TABLET_SERVER);
```

Comment:
Would be a follow on issue, would be nice to make this work w/ all the serer types.